packages/forklift-console-plugin/package.json

```diff
@@ -20,7 +20,7 @@
     "@migtools/lib-ui": "^8.4.1"
   },
   "peerDependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "0.0.17",
```

Comment:
should we update the polyfills to the new version?

Reply:
yes, the polyfills now pipe the new version unless we are running an old console, then it will fallback to the polyfill implementation

Reply:
yes